packages/webapp/src/__tests__/registryManager.test.ts

```
@@ -0,0 +1,452 @@
+		global.URL = class {
+			searchParams = {
+				set: vi.fn(),
+				delete: vi.fn(),
+				has: vi.fn(),
+				get: vi.fn()
+			};
+			toString() {
+				return 'http://test.com';
+			}
+			constructor() {
+				return this;
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)